crates/core/tedge_api/Cargo.toml

```diff
@@ -19,6 +19,8 @@ mqtt_channel = { workspace = true }
 serde = { workspace = true }
 serde_json = { workspace = true }
 shell-words = { workspace = true }
+# TODO this feels like a bad way to structure things
+tedge_config = { workspace = true }
```

Comment:
This feels like a weird artifact of putting the `TimeFormat` enum in `tedge_config` and it feels like `tedge_api` shouldn't depend on `tedge_config`. @didier-wenzek do you agree/have thoughts on how to resolve this?

Reply:
See https://github.com/thin-edge/thin-edge.io/pull/2508#discussion_r1423944331.

If moving formatting code to `tedge_config` raises other concerns, an alternative is to move the format definition and formatting code to `tedge_utils`, on which  `tedge_config`  already depends.